uiowa/uiowa
# [Registrar] Dates by Session

# How to test
1. Pull this branch
2. Go to https://registrar.uiowa.ddev.site/calendars-and-deadlines/academic-calendar
3. Edit the layout of the page and set the `Session(s) to display` to `Current session, plus next ten sessions` and also check the `Include past sessions` checkbox.
4. Notice that we are now displaying 10 sessions before the current session, and 10 sessions after, and all the subsessions in between.
5. Note as well that you can emulate the old [dates by session](https://registrar.uiowa.edu/dates-by-session) page and that this shows all the same information.
6. Check that subsessions show by default now, and that the toggle no longer shows on the user facing form.
7. Check that the session dropdown now has all subsessions indented to show that they are under the main seasonal sessions.
